communications/slack.md

```diff
@@ -0,0 +1,91 @@
+## On Slack
+
+Slack is a resourceful tool being used by a complex company. Each team will have specific dynamics that are from the scope of their tasks and it might require diverse communication tools.
+
+This is a gathering of some of the most used resources in slack to efficiently communicate based on takes from different people within the company. 
+
+Have in mind who you're trying to reach: If the person you want to correspond with is a highly contested resource, the expectation from them to engage with you synchronously will be frustrating. That's why we say `default to async communication`. 
+
+**Opt-in**: Join channels that you are interested in or obligated to.
+
+**Minimize Distractions**: Nowadays we have way too many channels of receiving information, by choice or otherwise. Instead of striving to achieve "Notification Zero", try to be as close as possible to "Everything Zero". This means diligently clear or defer any information that is not work-related. 
+
+Custom display channel: Star the channel heavily used, and only display unread channels if you prefer a clean interface. Click with the left button on the channel and select `Star channel`. 
+<p align="center">
+ <img width="200" alt="star" src=https://user-images.githubusercontent.com/53827690/89816335-63e4f680-db14-11ea-85dc-b42ce633a89a.png>
+ <img width="400" alt="display" src=https://user-images.githubusercontent.com/53827690/89817201-becb1d80-db15-11ea-9057-3fbd7ac038c0.png>
+</p>
+
+Use the sidebar section to organize channels in different use cases. For example, all dev related channels can be under a `Team` section to remind of the highest priority to read.
```

Comment:
on top of this, we can include on the use of Slack's official `/remind` slash command (it's fairly intuitive, and interactive)